recipes/openblas/all/conanfile.py

```diff
@@ -64,100 +78,108 @@ def validate(self):
         if hasattr(self, "settings_build") and cross_building(self, skip_x64_x86=True):
             raise ConanInvalidConfiguration("Cross-building not implemented")
 
-    def source(self):
-        get(self,
-            **self.conan_data["sources"][self.version],
-            strip_root=True,
-            destination=self.source_folder
-        )
+        if self.options.build_relapack:
+            if not self.options.build_lapack:
+                raise ConanInvalidConfiguration(f'"{self.name}/*:build_relapack=True" option requires "{self.name}/*:build_lapack=True"')
+            if self.settings.compiler not in ["gcc", "clang"]:
+                # ld: unknown option: --allow-multiple-definition on apple-clang
+                raise ConanInvalidConfiguration("build_relapack option is only supported for GCC and Clang")
```

Comment:
```suggestion
                raise ConanInvalidConfiguration(f'"{self.name}/*:build_relapack=True" option is only supported for GCC and Clang')
```

Reply:
Asking the team if this looks alright for them when the message is not asking the user to set the option by itself